test/screenshotter/ss_data.yaml

```diff
@@ -170,6 +170,7 @@ Symbols1: |
     \maltese\degree\pounds\$
     \text{\maltese\degree}
 Text: \frac{a}{b}\text{c~ {ab} \ e}+fg
+TextWithMath: \text{for $a < b$ and $ c < d $}.
```

Comment:
Sorry, yeah -- I can only run the screenshotter docker on a different machine, and pushing was the easiest way.  I'm rebasing now to merge in the images.